aeron-cluster/src/main/java/io/aeron/cluster/service/ClusterMarkFile.java

```diff
@@ -111,40 +115,70 @@ public final class ClusterMarkFile implements AutoCloseable
     private final int headerOffset;
 
     /**
-     * Create new {@link MarkFile} for a cluster service but check if an existing service is active.
+     * Create new {@link MarkFile} for a cluster component but check if an existing component is active.
      *
      * @param file              full qualified file to the {@link MarkFile}.
      * @param type              of cluster component the {@link MarkFile} represents.
      * @param errorBufferLength for storing the error log.
      * @param epochClock        for checking liveness against.
      * @param timeoutMs         for the activity check on an existing {@link MarkFile}.
+     * @deprecated Use {@link #ClusterMarkFile(File, ClusterComponentType, int, EpochClock, long, int)} instead.
      */
+    @Deprecated(forRemoval = true)
     public ClusterMarkFile(
         final File file,
         final ClusterComponentType type,
         final int errorBufferLength,
         final EpochClock epochClock,
         final long timeoutMs)
+    {
+        this(file, type, errorBufferLength, epochClock, timeoutMs, 0);
```

Comment:
I would be inclined to use `MIN_PAGE_SIZE` instead of `0` here.  This will remove the extra checks for 0 to default it to that value.  Are there any other places where we end up passing a `0` to the ClusterMarkFile constructor?

Reply:
I went with 4KB default across all impls.